src/styles/blocks/title.css

```diff
@@ -0,0 +1,21 @@
+.tittle1 {
+  font-weight: 500;
+  font-size: var(--font-mid);
+  line-height: var(--line-big);
+
+  color: var(--color-dark-blue);
+  margin-top: 40px;
+
+  margin-bottom: 0;
+}
+
+.tittle2 {
```

Comment:
seems like you have already defined such a class

Reply:
Same